samples/go/noms-merge/main_test.go

```diff
@@ -65,7 +65,7 @@ func (s *testSuite) TestWin() {
 	})
 
 	var mainErr error
-	stdout, stderr := s.Run(func() { mainErr = nomsMerge() }, []string{"--quiet=true", "--parent=" + p, s.LdbDir, l, r})
+	stdout, stderr, _ := s.Run(func() { mainErr = nomsMerge() }, []string{"--quiet=true", "--parent=" + p, s.LdbDir, l, r})
```

Comment:
Simply didn't want to muck around with the existing test logic too much in this commit. I think @cmasone-attic  has an issue for me to refactor test a little bit to mesh better with noms-merge